AndroidAssignment/app/src/main/java/org/sopt/androidassignment/SignInActivity.kt

```diff
@@ -0,0 +1,46 @@
+package org.sopt.androidassignment
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Toast
+import org.sopt.androidassignment.databinding.ActivitySignInBinding
+
+class SignInActivity : AppCompatActivity() {
+
+    private lateinit var binding : ActivitySignInBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        binding = ActivitySignInBinding.inflate(layoutInflater)
+
+        val intent = Intent(this, HomeActivity::class.java)
+        binding.btLogin.setOnClickListener{
+            var str = binding.etId.getText()
+            var pw = binding.etPw.getText()
```

Comment:
코틀린의 경우 프로퍼티라는 기능을 제공하기때문에 이렇게만 표현하셔도 됩니다~!!
```suggestion
              val str = binding.etId.text
            val pw = binding.etPw.text
```
그리고 var의 경우 변수가 변할 가능성이 있을 때 쓰는 것이니 현재 불변하는 변수인 str과 pw는 val로 정의해도 괜찮을 것 같아요~!!